src/cli/telemetry/test/telemetry.spec.ts

```diff
@@ -137,10 +139,10 @@ describe('prepareData', () => {
   const sys = createSystem();
 
   it('provides an object', async () => {
-    const data = await telemetry.prepareData({}, config, sys, 1000);
+    const data = await telemetry.prepareData(coreCompiler, config, sys, 1000);
     expect(data).toEqual({
       arguments: [],
-      build: 'unknown',
+      build: coreCompiler.buildId,
```

Comment:
these changes are just consequences of calling these functions with an actual `coreCompiler` instance instead of `{}`